v3io/dataplane/transport/httpclient.py

```diff
@@ -100,6 +100,9 @@ def wait_response(self, request, raise_for_status=None, num_retries=1):
 
                 # re-send the request on the connection
                 request = self._send_request_on_connection(request, connection)
+            except v3io.dataplane.response.HttpResponseError as response_error:
+                self._logger.warn_with('Response error: {}'.format(str(response_error)))
+                raise response_error
```

Comment:
@roybabayov I think you should have used `raise` instead of `raise response_error`, otherwise you'll show the wrong stack trace